lib/public/components/qcFlags/qcFlagsPagesButtons.js

```diff
@@ -58,22 +58,40 @@ export const qcFlagCreationPanelLink = (
         ),
         Success: (detectors) => {
             const userHasAccessToDetector = detectors.find(({ id }) => id === dplDetectorId);
+
             if (userHasAccessToDetector) {
                 return remoteRun.match({
                     NotAsked: () => null,
                     Loading: () => null,
                     Failure: () => tooltip(h('.f3', iconWarning()), 'An error has occurred. Unable to determine whether run is subject for QC'),
                     Success: (run) => isRunNotSubjectToQc(run)
                         ? h('button.btn.btn-primary', { disabled: true, title: getRunNotSubjectToQcReason(run) }, content)
-                        : frontLink(
-                            content,
-                            pageName,
-                            { runNumber: run.runNumber, dplDetectorId, ...options },
-                            {
-                                class: ['btn', 'btn-primary', ...linkClasses].join(' '),
-                                title: 'Create a QC flag',
-                            },
-                        ),
+                        : h('div.qc-flag-container', [
+                            h('style', `
+                                .qc-flag-container { display: flex; align-items: center; gap: 8px; padding-right: 5px }
+                                .qc-link-wrapper { pointer-events: auto; }
+                                .select-multi-flag { cursor: pointer; transform: scale(1.5); pointer-events: auto; }
+                            `),
```

Comment:
Why all that?
Always use stylesheet, and try to use as much as possible existing classes.